### ISSUE TITLE
[Feature]: allow disabling crashpad process

### ISSUE THREAD

**Issue by pushkin-** [MicrosoftEdge/WebView2Feedback#4966]
### Describe the feature/enhancement you need

somewhat related to this issue https://github.com/MicrosoftEdge/WebView2Feedback/issues/4965

Setting that flag to true or false doesn't disable the crashpad-handler process.

Electron somehow manages to disable it and only runs it when crash reporting is running.

Would be nice to turn it off when we don't need it.

### The scenario/use case where you would use this feature

remove extra process, save memory

### How important is this request to you?

Nice to have. There are other ways to tackle this, but having official API support would be beneficial.

### Suggested implementation

I feel like that [`IsCustomCrashReportingEnabled`](https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2environmentoptions.iscustomcrashreportingenabled?view=webview2-dotnet-1.0.2849.39) property should control whether that process runs.

**Comment by aluhrs13** [comment]
I don't think this is something we'd strongly consider. Crashpad's memory usage is pretty much in the single digit MB, often <1MB. If we did, it wouldn't be tied to `IsCustomCrashReportingEnabled` (or would be a breaking change) - that's intended for the app to say "I'm handling the reporting of my crashes myself" which may or may not use the crash dumps gathered by WV2.
